validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandler.java

```diff
@@ -284,6 +288,34 @@ public void sendSignedAttestation(
         () -> apiClient.sendSignedBlock(new tech.pegasys.teku.api.schema.SignedBeaconBlock(block)));
   }
 
+  @Override
+  public SafeFuture<Optional<SubmitCommitteeSignaturesResult>> sendSyncCommitteeSignatures(
+      final List<SyncCommitteeSignature> syncCommitteeSignatures) {
+    return sendRequest(
+        () ->
+            apiClient
+                .sendSyncCommitteeSignatures(
+                    syncCommitteeSignatures.stream()
+                        .map(
+                            i ->
```

Comment:
nit: I'd probably call this `signature` instead of `I`.

Reply:
i got sick of typing nice names, but i'll fix.